### PR TITLE
HDDS-2745. shell/KeyCommands.md translation

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

one .md file

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2745

## How was this patch tested?

hugo server -D

### CODE REVIEW

File: hadoop-hdds/docs/content/shell/KeyCommands.zh.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+### 列举
+
+用户通过 `key list` 命令列出一个桶中的所有键。
+
+***参数：***
+
+| 参数名                      |  说明                                |
+|--------------------------------|-----------------------------------------|
+| -l, \-\-length                   | 返回结果的最大数量，默认值为 1000
+| -p, \-\-prefix                   | 可选，只有匹配指定前缀的键会被返回
```
Comment (iamabug) [inline hadoop-hdds/docs/content/shell/KeyCommands.zh.md]: original doc says,

```
Optional, Only buckets that match this prefix will be returned.
```

I guess it should be

```
Optional, Only keys that match this prefix will be returned.
```

Am I right ?

**Comment by iamabug** [comment]
@xiaoyuyao Could you help review this ? Thanks

File: hadoop-hdds/docs/content/shell/KeyCommands.zh.md (hunk 1 above)
Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/shell/KeyCommands.zh.md]: Good catch. Please fix the English document as well.

File: hadoop-hdds/docs/content/shell/KeyCommands.zh.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
+---
+title: 键命令
+summary: 键命令帮助你管理键和对象的生命周期
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/shell/KeyCommands.zh.md]: 键和对象=>键/对象

File: hadoop-hdds/docs/content/shell/KeyCommands.zh.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+### 列举
+
+用户通过 `key list` 命令列出一个桶中的所有键。
+
+***参数：***
+
+| 参数名                      |  说明                                |
+|--------------------------------|-----------------------------------------|
+| -l, \-\-length                   | 返回结果的最大数量，默认值为 1000
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/shell/KeyCommands.zh.md]: 默认值为 1000=>默认值为 100

If the English version has 1000, that needs to be fixed too.

File: hadoop-hdds/docs/content/shell/KeyCommands.zh.md (hunk 1 above)
Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/shell/KeyCommands.zh.md]: Yes, Please fix the English Document too.

**Comment by xiaoyuyao** [comment]
LGTM, +1. I will merge it shortly.

**Pull request merged**